external/test-runtime/XUnit.Runtime.depproj

```diff
@@ -81,7 +81,7 @@
       <Version>1.0.0-prerelease</Version>
     </PackageReference>
     <PackageReference Include="System.Drawing.Common.TestData">
-      <Version>1.0.6</Version>
+      <Version>1.0.7</Version>
```

Comment:
Why is the Xunit update needed?

Reply:
It is to get the newest drawing-testsdata that contains an icon that reproduces this issue before the fix and added a test for it as well.